src/toil/test/docs/scripts/tutorial_wdlexample.py

```diff
@@ -25,7 +27,9 @@ def runQC(job, wdl_file, wdl_filename, json_file, json_filename, outputs_dir, ja
 
 
 if __name__ == "__main__":
-    options = Job.Runner.getDefaultOptions("./toilWorkflowRun")
+    jobstore: str = tempfile.mkdtemp("tutorial_wdlexample")
+    os.rmdir(jobstore)
```

Comment:
Is this still safe from known-temporary-file-name attacks? We don't want to show something vulnerable in a tutorial.

Reply:
Yep, it is secure!

https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp

> Creates a temporary directory in the most secure manner possible. There are no race conditions in the directory’s creation. The directory is readable, writable, and searchable only by the creating user ID.

Reply:
It's not secure after the directory is removed, though, right? Because someone else could re-create something at that name before Toil can.

Reply:
True, it would be better if the job store directory was allowed to exist (if and only if it was empty already)